### ISSUE TITLE
Implementation for Kotlin (& Java)

### ISSUE THREAD

**Issue by petersamokhin** [kjk/notionapi#10]
Hello!
There are Kotlin implementation: https://github.com/petersamokhin/knotion-api

If you want, you can add it to the end of readme.

Thanks for the library!

P.S. used to write it because your library isn't working, don't know why, maybe API implementation is out of date.

**Comment by kjk** [comment]
Could you provide more details about what is not working?

I just tested with `.\s\tohtml.ps1 4c6a54c68b3e4ea2af9cfaabcc88d58d` and it downloaded the page and converted to html as expected.

**Comment by petersamokhin** [comment]
I tried this code:

```
client := &notionapi.Client{}
client.AuthToken = token
page, err := client.DownloadPage(paidBoostersPageId)
if err != nil {
	fmt.Printf("DownloadPage() failed with %s\n", err)
}

fmt.Printf("%+v\n", page.Tables)
```

But there are empty array and `[]` output, with correct token and page id.
Then I tried to reverse notion's API in chrome and wrote own wrapper, so there are no problems in my implementation not.
I think, maybe some field names are changed for now and JSON unmarshalling isn't working in your library, but only `Users`, `Root` and `ID` fields are available from `page`.
